AltUseVendors/__init__.py

```diff
@@ -73,6 +73,29 @@ class ShopInfo:
 VIAL_HEAL_PERCENT: float = 0.25
 
 
+# Sound AkEvents for playing
+AKE_BUY: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Buy"
+AKE_SELL: str = "Ake_UI.UI_Vending.Ak_Play_UI_Vending_Sell"
+AKE_DICT = {
```

Comment:
Buy and sell are reasonable enough names, but what's 'dict' supposed to mean? Prefer something like `AKE_PURCHASE_BY_VENDOR`.

You could do the game lookup here once at the root level, rather than needing to do so each time, since it'll never change.
```py
AKE_PURCHASE_BY_VENDOR = {
  Game.BL2: ...
}[Game.GetCurrent()]
```

Reply:
Changed to AKE_VOICELINE_BY_VENDOR and looked up the game like you have there.